mqt/getaddons.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
```

Comment:
Could just do as https://github.com/OCA/maintainer-quality-tools/blob/master/git/getaddons.py?

This way the file is unique.

And also do for the other copied files.